hugo/content/3_how-tos/31_account-management/309_user-management.md

```diff
@@ -75,22 +75,17 @@ To add/remove users from the project:
 
 1. On the left navigation bar, click **Account > Project**. The panel displays a list of projects.
 
-2. Select the project to which users need to be added/removed.
+2. Under **Actions**, select the **Edit** icon.
 
 3. To add users:
-   
-   1. On **Users** tab, enter a valid email address of the user you want to invite to the project.
 
-   2. Click **ADD USER**.
+   1. Select the **Users** tab, enter a valid email address of the user you want to invite to the project.
```

Comment:
**enter a valid email address of the user you want to invite to the project.** is not required

Reply:
@SanchithaSridhar this step is necessary, will change it to "Select the Users tab, where you can use the search field to search for the user by username or email address."